src/arweave/composite-client.ts

```diff
@@ -256,6 +261,8 @@ export class ArweaveCompositeClient
   }
 
   async refreshPeers(): Promise<void> {
+    this.log.debug('ArweaveCompositeClient.refreshPeers refresh called');
```

Comment:
Fine to leave this, but let's create a child logger with the method name and change the message to "Refreshing peers...".

Reply:
done